Sources/Subs-List.php

```diff
@@ -73,11 +79,18 @@ function createList($listOptions)
 	else
 	{
 		// First get an impression of how many items to expect.
-		if (isset($listOptions['get_count']['file']))
-			require_once($listOptions['get_count']['file']);
+		if (!empty($listOptions['get_count']['value']) && ctype_digit($listOptions['get_count']['value']))
```

Comment:
I smell a bug. This won't work with integers that I know of.

Reply:
Which integers do you know that won't get caught by ctype_digit?